Data/equity/usa/factor_files/aig.csv

```diff
@@ -1,69 +1,69 @@
-19980102,0.7224214,7.11238
-19980303,0.7224214,7.11238
-19980602,0.7228660,7.11238
-19980731,0.7233058,7.11238
-19980901,0.7233058,10.661
-19981201,0.7238269,10.661
-19990302,0.7242731,10.661
-19990601,0.7246296,10.661
-19990730,0.7249906,10.661
-19990831,0.7249906,13.3333
-19991129,0.7253816,13.3333
-20000229,0.7257351,13.3333
-20000530,0.7261453,13.3333
-20000728,0.7264628,13.3333
-20000829,0.7264628,20
-20001128,0.7267746,20
-20010227,0.7270617,20
-20010529,0.7273796,20
-20010806,0.7277071,20
-20011204,0.7280821,20
-20020226,0.7284550,20
-20020604,0.7288770,20
-20020903,0.7293518,20
-20021203,0.7299270,20
-20030304,0.7304734,20
-20030603,0.7311958,20
-20030902,0.7317900,20
-20031202,0.7325966,20
-20040302,0.7334144,20
-20040601,0.7340596,20
-20040831,0.7347136,20
-20041130,0.7354880,20
-20050301,0.7363598,20
-20050531,0.7377453,20
-20050830,0.7394090,20
-20051129,0.7412845,20
-20060228,0.7429219,20
-20060530,0.7446047,20
-20060829,0.7464522,20
-20061128,0.7483935,20
-20070227,0.7501655,20
-20070529,0.7520238,20
-20070904,0.7537537,20
-20071204,0.7560256,20
-20080304,0.7587525,20
-20080603,0.7620782,20
-20080902,0.7663345,20
-20090630,0.7740893,20
-20110119,0.7740893,1
-20130909,0.9270896,1
-20131202,0.9289785,1
-20140306,0.9308511,1
-20140605,0.9331262,1
-20140908,0.9352542,1
-20141201,0.9373726,1
-20150309,0.9396172,1
-20150608,0.9417154,1
-20150909,0.9436929,1
-20151202,0.9482275,1
-20160309,0.9524275,1
-20160608,0.9583721,1
-20160912,0.9637905,1
-20161205,0.9690481,1
-20170310,0.9739018,1
-20170609,0.9788659,1
-20170913,0.9837718,1
-20171206,0.9889812,1
-20180313,0.9942806,1
-20501231,1.0000000,1
+19980102,0.7224214,7.112380,110.25
+19980303,0.7224214,7.112380,120.25
+19980602,0.7228660,7.112380,122.69
+19980731,0.7233058,7.112380,151.13
+19980901,0.7233058,10.66100,77.38
+19981201,0.7238269,10.66100,94.13
+19990302,0.7242731,10.66100,112.88
+19990601,0.7246296,10.66100,113.94
+19990730,0.7249906,10.66100,118
+19990831,0.7249906,13.333300,92.69
+19991129,0.7253816,13.333300,105.65
+20000229,0.7257351,13.333300,87.19
+20000530,0.7261453,13.333300,113.25
+20000728,0.7264628,13.333300,129.56
```

Comment:
Possibly should be normalized?

Reply:
I'll update the generator to normalize and refresh these files.

Reply:
I noticed that *some* of the entries had trailing zeros, either way it's nice to not have them, saves on space :)